api/tests/test_cloud_record_summary.py

```diff
@@ -36,10 +37,14 @@ def test_cloud_record_summary_get_IAM_fail(self):
                                            "Day",
                                            "Month",
                                            "Year"]):
+
             test_client = Client()
-            response = test_client.get('/api/v1/cloud/record/summary?'
-                                       'group=TestGroup&'
-                                       'from=20000101&to=20191231',
+            get_url = ''.join((reverse('CloudRecordSummaryView'),
```

Comment:
It was separated out because this one is more indented than most of the others due to being in a `with` block and pep8 complains if it's wrapped up like line 113.

I'll split them all out to separate variables and change the name.